common/hsm_version.h

```diff
@@ -27,6 +27,7 @@
  * v5 with preapprove_check: 0ed6dd4ea2c02b67c51b1420b3d07ab2227a4c06ce7e2942d946967687e9baf7
  * v6 no secret from get_per_commitment_point: 0cad1790beb3473d64355f4cb4f64daa80c28c8a241998b7ef0223385d7ffff9
  * v6 with sign_bolt12_2 (tweak using node id): 8fcb731279a10af3f95aeb8be1da6b2ced76a1984afa18c5f46a03515d70ea0e
+ * v6 with sign_message_with_key: e919c58fb80be639feb8502a740f0ed80a87c5885154c5632a0dacf2d96b9899
```

Comment:
Isn't V6 already released? This is a breaking change and we need to ask the signer to be at least V7, i.e. the first version that implements all expected API s.

Reply:
Thanks. I had no idea...